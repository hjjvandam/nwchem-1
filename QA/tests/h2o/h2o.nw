start h2o

scratch_dir    /scratch
permanent_dir  /scratch

geometry units au
 O 0       0        0
 H 0       1.430   -1.107
 H 0      -1.430   -1.107
end


basis
  O library 6-31g*
  H library 6-31g*
end

task scf gradient
