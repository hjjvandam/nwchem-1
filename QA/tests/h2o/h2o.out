 argument  1 = h2o
  Unable to open h2o --- appending .nw
                                         
                                         
 
 
              Northwest Computational Chemistry Package (NWChem) 2.1
              ------------------------------------------------------
 
 
                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352
 
                                         
                                         
 
 
                         COPYRIGHT (C) 1994, 1995, 1996, 1997
                        Pacific Northwest National Laboratory,
                             Battelle Memorial Institute.
 
                            >>> All Rights Reserved <<<
 
 
                                    DISCLAIMER
                                    ----------

            This material was prepared as an account of work sponsored
            by an agency of the United States Government.  Neither the
            United States Government nor the United States Department
            of Energy, nor Battelle, nor any of their employees, MAKES
            ANY WARRANTY, EXPRESS OR IMPLIED, OR ASSUMES ANY LEGAL
            LIABILITY OR RESPONSIBILITY FOR THE ACCURACY, COMPLETENESS,
            OR USEFULNESS OF ANY INFORMATION, APPARATUS, PRODUCT,
            SOFTWARE, OR PROCESS DISCLOSED, OR REPRESENTS THAT ITS USE
            WOULD NOT INFRINGE PRIVATELY OWNED RIGHTS.


                                    LIMITED USE
                                    -----------

            This software (including any documentation) is being made
            available to you for your internal use only, solely for use
            in performance of work directly for the U.S. Federal
            Government or work under contracts with the U.S. Department
            of Energy or other U.S. Federal Government agencies.  This
            software is a version which has not yet been evaluated and
            cleared for commercialization.  Adherence to this notice
            may be necessary for the author, Battelle Memorial
            Institute, to successfully assert copyright in and
            commercialize this software. This software is not intended
            for duplication or distribution to third parties without
            the permission of the Manager of Software Products at
            Pacific Northwest National Laboratory, Richland,
            Washington, 99352.


                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were produced with
            Government support under Contract Number DE-AC06-76RLO-1830
            awarded by the United States Department of Energy.  The
            Government retains a paid-up non-exclusive, irrevocable
            worldwide license to reproduce, prepare derivative works,
            perform publicly and display publicly by or for the
            Government, including the right to distribute to other
            Government contractors.


              Job information
              ---------------

    hostname  = fermi
    program   = /msrc/home/d3g681/nwchem/bin/SOLARIS/nwchem
    date      = Mon Jun 23 11:26:54 1997

    compiled  = Sun Jun 22 12:44:21 PDT 1997
    source    = /msrc/home/d3g681/nwchem
    input     = h2o.nw
    prefix    = h2o.
    data base = /scratch/h2o.db
    status    = startup
    nproc     =    1


            Memory information
            ------------------

    heap      =    1572865 doubles
    stack     =    1572865 doubles
    global    =    3145728 doubles (allocated separately from heap & stack)
    total     =    6291458 doubles
    verify    = yes
    hardfail  = no 


           Directory information
           ---------------------
 
  0 permanent = /scratch
  0 scratch   = /scratch
 
 
 
                                NWCHEM Input Module
                                -------------------
 
 
 
 
                             Geometry "geometry" -> ""
                             -------------------------
 
 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)
 
  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000     0.22140000
    2 H                    1.0000     0.00000000     1.43000000    -0.88560000
    3 H                    1.0000     0.00000000    -1.43000000    -0.88560000
 
      Atomic Mass 
      ----------- 
 
      O                 15.994910
      H                  1.007825
 

 Effective nuclear repulsion energy (a.u.)       9.1971984402

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000
 
                      Basis "ao basis" -> "" (cartesian)
                      -----
  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S     5484.671700  0.001831
  1 S      825.234950  0.013950
  1 S      188.046960  0.068445
  1 S       52.964500  0.232714
  1 S       16.897570  0.470193
  1 S        5.799635  0.358521
 
  2 S       15.539616 -0.110778
  2 S        3.599934 -0.148026
  2 S        1.013762  1.130767
 
  3 P       15.539616  0.070874
  3 P        3.599934  0.339753
  3 P        1.013762  0.727159
 
  4 S        0.270006  1.000000
 
  5 P        0.270006  1.000000
 
  6 D        0.800000  1.000000
 
  H (Hydrogen)
  ------------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S       18.731137  0.033495
  1 S        2.825394  0.234727
  1 S        0.640122  0.813757
 
  2 S        0.161278  1.000000
 


 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31g*                  6       15   3s2p1d
 H                           6-31g*                  2        2   2s


                                 NWChem SCF Module
                                 -----------------
 
 

  ao basis        = "ao basis"
  functions       =    19
  atoms           =     3
  closed shells   =     5
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = /scratch/h2o.movecs
  use symmetry    = F
  symmetry adapt  = F


 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 O                           6-31g*                  6       15   3s2p1d
 H                           6-31g*                  2        2   2s



 Forming initial guess at       1.5s

  Calling module = scf
 
      Superposition of Atomic Density Guess
      -------------------------------------
 
 Sum of atomic energies:         -75.75081731

 #quartets = 1.540D+03 #integrals = 8.874D+03 #direct =  0.0% #cached =100.0%


 Integral file          = /scratch/h2o.aoints.0
 Record size in doubles =  32769        No. of integs per rec  =  32768
 Max. records in memory =      2        Max. records in file   =  10985
 No. of bits per label  =      8        No. of bits per value  =     32

 
      Non-variational initial energy
      ------------------------------

 Total energy =     -75.919952
 1-e energy   =    -121.737767
 2-e energy   =      36.620616
 HOMO         =      -0.470482
 LUMO         =       0.114886
 

 Starting SCF solution at       2.2s



 ----------------------------------------------
         Quadratically convergent ROHF

 Convergence threshold     :          1.000E-04
 Maximum no. of iterations :           20
 Final Fock-matrix accuracy:          1.000E-07
 ----------------------------------------------


              iter       energy          gnorm     gmax       time
             ----- ------------------- --------- --------- --------
                 1      -75.9747705736  8.31D-01  3.41D-01      2.1
                 2      -76.0080998440  1.84D-01  1.02D-01      2.2
                 3      -76.0104204930  4.08D-02  2.40D-02      2.2
                 4      -76.0105383805  1.63D-03  9.29D-04      2.3
                 5      -76.0105386194  2.29D-06  9.65D-07      2.4


       Final RHF  results 
       ------------------ 

         Total SCF energy =    -76.010538619412
      One-electron energy =   -123.058841715246
      Two-electron energy =     37.851104655636
 Nuclear repulsion energy =      9.197198440198

        Time for solution =      0.3s


             Final eigenvalues
             -----------------

              1      
    1  -20.5603
    2   -1.3419
    3   -0.7071
    4   -0.5711
    5   -0.4979
    6    0.2108
    7    0.3042
    8    1.0227
    9    1.1318
   10    1.1678
   11    1.1719
   12    1.3809
   13    1.4341
   14    2.0201
   15    2.0337
 
                       ROHF Final Molecular Orbital Analysis
                       -------------------------------------
 
 Vector    2  Occ=2.000000D+00  E=-1.341930D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      0.475861  1 O  s                  6      0.439200  1 O  s          
     1     -0.209676  1 O  s          
 
 Vector    3  Occ=2.000000D+00  E=-7.070590D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      0.507652  1 O  py                 8      0.306488  1 O  py         
    18     -0.230979  3 H  s                 16      0.230979  2 H  s          
 
 Vector    4  Occ=2.000000D+00  E=-5.710706D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.555445  1 O  pz                 9      0.403175  1 O  pz         
     6      0.325537  1 O  s                  2      0.164592  1 O  s          
 
 Vector    5  Occ=2.000000D+00  E=-4.979252D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      0.639616  1 O  px                 7      0.511469  1 O  px         
 
 Vector    6  Occ=0.000000D+00  E= 2.108019D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.415543  1 O  s                 19     -1.041448  3 H  s          
    17     -1.041448  2 H  s                  9     -0.508219  1 O  pz         
     5     -0.217055  1 O  pz         
 
 Vector    7  Occ=0.000000D+00  E= 3.042324D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      1.395012  2 H  s                 19     -1.395012  3 H  s          
     8     -0.833804  1 O  py                 4     -0.329272  1 O  py         
 
 Vector    8  Occ=0.000000D+00  E= 1.022734D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      0.838912  2 H  s                 18     -0.838912  3 H  s          
     8     -0.662617  1 O  py                17     -0.459237  2 H  s          
    19      0.459237  3 H  s                 14     -0.343167  1 O  dyz        
 
 Vector    9  Occ=0.000000D+00  E= 1.131842D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      1.636695  1 O  s                  2     -0.921950  1 O  s          
     9      0.708674  1 O  pz                18      0.548806  3 H  s          
    16      0.548806  2 H  s                 19     -0.474054  3 H  s          
    17     -0.474054  2 H  s                  5     -0.419234  1 O  pz         
    10     -0.387487  1 O  dxx               15     -0.318052  1 O  dzz        
 
 Vector   10  Occ=0.000000D+00  E= 1.167786D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      1.036049  1 O  px                 3     -0.962745  1 O  px         
 
 Vector   11  Occ=0.000000D+00  E= 1.171944D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      0.761011  1 O  pz                18      0.661200  3 H  s          
    16      0.661200  2 H  s                  6     -0.462713  1 O  s          
     9     -0.370411  1 O  pz                19     -0.357348  3 H  s          
    17     -0.357348  2 H  s                 13      0.249351  1 O  dyy        
 
 Vector   12  Occ=0.000000D+00  E= 1.380936D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      1.537091  1 O  py                 4     -1.037375  1 O  py         
    17     -0.915090  2 H  s                 19      0.915090  3 H  s          
 
 Vector   13  Occ=0.000000D+00  E= 1.434077D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      3.574399  1 O  s                  2     -1.418425  1 O  s          
     9     -1.174649  1 O  pz                19     -0.784923  3 H  s          
    17     -0.784923  2 H  s                 13     -0.644454  1 O  dyy        
     5      0.506383  1 O  pz                15     -0.402327  1 O  dzz        
    16     -0.323132  2 H  s                 18     -0.323132  3 H  s          
 
 Vector   14  Occ=0.000000D+00  E= 2.020054D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      1.008815  1 O  dzz               13     -0.580615  1 O  dyy        
    10     -0.366779  1 O  dxx                6     -0.227333  1 O  s          
 
 Vector   15  Occ=0.000000D+00  E= 2.033721D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    11      1.732051  1 O  dxy        
 
 
  Mulliken analysis of the total density
  --------------------------------------

    Atom       Charge   Shell Charges
 -----------   ------   -------------------------------------------------------
    1 O    8     8.87   2.00  0.90  2.90  0.92  2.07  0.08
    2 H    1     0.57   0.46  0.10
    3 H    1     0.57   0.46  0.10
 
     Multipole analysis of the density
     ---------------------------------
 
     L   x y z        total         open         nuclear
     -   - - -        -----         ----         -------
     0   0 0 0      0.000000      0.000000     10.000000
 
     1   1 0 0      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000
     1   0 0 1     -0.875294      0.000000      0.000000
 
     2   2 0 0     -5.372335      0.000000      0.000000
     2   1 1 0      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000
     2   0 2 0     -3.071804      0.000000      4.089800
     2   0 1 1      0.000000      0.000000      0.000000
     2   0 0 2     -4.444599      0.000000      1.960718
 

 Parallel integral file used       1 records with       0 large values

                              NWCHEM GRADIENTS Module
                              -----------------------
 
 

  wavefunction    =   RHF     


 Read molecular orbitals from /scratch/h2o.movecs

  numq   1540


                         RHF ENERGY GRADIENTS

    atom               coordinates                        gradient
                 x          y          z           x          y          z
   1 O       0.000000   0.000000   0.221400    0.000000   0.000000   0.014490
   2 H       0.000000   1.430000  -0.885600    0.000000   0.007296  -0.007245
   3 H       0.000000  -1.430000  -0.885600    0.000000  -0.007296  -0.007245
 
                 ----------------------------------------
                 |  Time  |  1-e(secs)   |  2-e(secs)   |
                 ----------------------------------------
                 |  CPU   |       0.02   |       2.27   |
                 ----------------------------------------
                 |  WALL  |       0.02   |       2.74   |
                 ----------------------------------------
 
 
                                NWCHEM Input Module
                                -------------------
 
 

 Summary of allocated global arrays 
 ----------------------------------

   No active global arrays
  

                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  223      223     2883      534     1340        1        0        0     
bytes total:             8.22e+05 3.13e+05 3.26e+05 4.00e+01 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 39432 bytes
 
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        15	        28
	current total bytes		         0	         0
	maximum total bytes		    532112	  18226612
	maximum total K-bytes		       533	     18227
	maximum total M-bytes		         1	        19



                                     CITATION
                                     --------

            Please use the following citation when publishing results
            obtained with NWChem:

            High Performance Computational Chemistry Group, "NWChem, A
            Computational Chemistry Package for Parallel Computers, 
            Version 2.1" (1997), Pacific Northwest National Laboratory,
            Richland, Washington 99352-0999, USA.



 Total times  cpu:        5.4s     wall:        6.1s
